theme/less/_sidebar.less

```diff
@@ -0,0 +1,65 @@
+#sidebar
+{
+  font-weight: bold;
+  font-size: 14px;
```

Comment:
I did notice that we are using precise pixels in some places and relative em/rem in other places, thinking we might need to adjust in future, to make sizing relative which might help with specifying variables.

I have created this -> https://github.com/browser-automation/cba-website/issues/28 

I guess we will tackle that at one point, but it shouldn't be blocker for this issue I guess.